src/App.tsx

```diff
@@ -15,71 +21,81 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends Component<{}, State> {
+  state = {
+    selectedGood: 'Jam',
+  };
 
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  render() {
+    const { selectedGood } = this.state;
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+    const clearSelection = () => (
+      this.setState({ selectedGood: '' })
+    );
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+    return (
+      <main className="section container">
+        {selectedGood
+          ? (
+            <h1 className="title is-flex is-align-items-center">
+              {`${selectedGood} is selected`}
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
Do you need this?)
```suggestion
```

Reply:
yes, because this line was originally in the code

Reply:
& Do you block linter for some reason?

Reply:
please re-read my comment carefully, this line was already in the code, it was not added by me personally.

and the task is different.

Please don't look at my repositories, I didn't ask you to.

Reply:
Yeah, I saw your comment) But, you are a feature frontEnd developer, if you know that you have some extra in your code - do you just leave it? 
It's strange that you so aggressive with me 😢

Reply:
I am not aggressive. I just don't know you, that's why your comments alerted me.

Reply:
I'm Serhii, graduated from mate academy, let's get to know each other :D

Reply:
pleased to meet you

Reply:
Nice to meet you too)
Everything is ok?